src/components/Loader/Loader.jsx

```diff
@@ -1,23 +1,40 @@
 // @flow
 import React from 'react';
+import cn from 'classnames';
 import style from './style.scss';
 import { getDataAttrs } from '../../dataUtils';
 import type { Data } from '../../types';
 
 type Props = {|
+  className?: string,
+  dotColor?: string,
+  dotClassName?: string,
   style?: Object,
-  dotStyle?: ?Object,
+  dotStyle: Object,
```

Comment:
can we use `{ [string]: string }` here since `Object` is generally unsafer?

Reply:
why not `StyleAttributes`? :)

Reply:
perfect!